restroom/.env.example

```diff
@@ -1,6 +1,8 @@
 FILES_DIR=../
 ZENCODE_DIR=../api
+LOGGER_DIR=../
```

Comment:
this should be absolute. Unrelated to this commit, but I think also other paths should be forced as absolute.